docs/metrics-details.md

```diff
@@ -12,7 +12,40 @@ This metrics can be used to be included into a monitoring platform like [**Grafa
 
 ## RIG Specific Metrics
 
-Currently there are no RIG specific metrics implemented yet. We are exposing the standard metrics providing information about the running application itself.
+### Proxy Metrics - `rig_proxy_requests_total` ###
+
+With this metric the usage of RIGs API Proxy/Gateway can be monitored.
+
+Following Labels are provided:
+- **method** - The HTTP method called for this request Following possible values:
+  - `GET`
+  - `POST`
+  - `PUT`
+  - `PATCH`
+  - `DELETE`
+  - `HEAD`
+  - `OPTIONS`
+- **path** - The path used for this request.
+- **target** - The target for this request. Grabbed from proxy parametrization. Following possible values:
+  - `http`
+  - `kafka`
+  - `kinesis`
+  - `N/A` - Not available, due to missing parametrization
+- **response_from** - Where the response is provided from. Grabbed from proxy parametrization. Following possible values:
```

Comment:
imho "configuration", "config" or "settings" is a bit clearer/simpler than "parametrization"